### ISSUE TITLE
Error executing apt-get update on the 5.2.2 image

### ISSUE THREAD

**Issue by Insanityisnice** [elastic/logstash-docker#20]
I get the following error when building out a custom image from 5.2.2.   I need to install a jdbc driver and got the following error when running the below file defined.   The issue is that in the user is changed to "logstash".  Easy enough to work around but very untypical for docker containers.  I should be able to setup extend the image with additional drivers easily.

W: chmod 0700 of directory /var/lib/apt/lists/partial failed - SetupAPTPartialDirectory (1: Operation not permitted)
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?

### Begin dockerfile  ###
FROM docker.elastic.co/logstash/logstash:5.2.2

RUN apt-get -y update \
 && apt-get install -y wget
### End dockerfile ###

To fix the issue change the user back to root in the docker file.  I think this should be done in the logstash image though to keep with convention.

**Comment by jarpy** [comment]
We are very keen to keep the container process running as a non-root user.

Would you consider doing:

```Dockerfile
FROM docker.elastic.co/logstash/logstash:5.2.2

USER root
RUN apt-get -y update && apt-get install -y wget
USER logstash
```
?

**Comment by jarpy** [comment]
I should mention that we change away from the root user in accordance with the [best practices document](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#user) from Docker Inc.

It could be that a lot of containers run as root by default, but not all processes need that level of privilege. Logstash doesn't need it, so we comply with security best practice and run as a regular user.

**Comment by Insanityisnice** [comment]
Changing users is the workaround I used.

It maybe that the rest of the container world is behind.  The other docker containers from elastic.co are not doing this and this is the first one that I've found that does.   I completely agree with not running as root it was just unexpended based on the other offerings.   It also presents an issue if your unaware given that sudo is not available in the default docker images.

If nothing else hopefully this will help someone else that hit's it unexpectedly.
